features-sdk/step_definitions/security-steps.js

```diff
@@ -1,24 +1,47 @@
 const
+  should = require('should'),
   {
-    Given
+    Given,
+    Then
   } = require('cucumber');
 
-Given('I create a profile {string} with the following policies:', function (profileId, dataTable) {
+Given('I create a profile {string} with the following policies:', async function (profileId, dataTable) {
   let policies = this.parseObject(dataTable);
+  return await this.sdk.security.createProfile(profileId, policies);
```

Comment:
No need to return something since the method will throw if the promise is rejected.

Also the result of this API call should be stored in `this.props.result` for further usage (apply for every step definition)